integration-test-apps/auto-instrumentation/flask/application.py

```diff
@@ -1,24 +1,34 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT-0
 
-# OTel Imports
-
-from opentelemetry import propagate
-
-# AWS X-Ray SDK Extension Components
-
-from opentelemetry.sdk.extension.aws.trace.propagation.aws_xray_format import (
-    AwsXRayFormat,
-)
+# Configuration with auto-instrumentation is done using environment variables.
+# See the following variables:
+# - OTEL_PROPAGATORS=xray
+# - OTEL_PYTHON_ID_GENERATOR=xray
+# - OTEL_RESOURCE_ATTRIBUTES='service.name=aws-sample-auto-app'
+# (These are the default values if using `opentelemetry-distro[otlp]`)
+# - OTEL_TRACES_EXPORTER=otlp
+# - OTEL_EXPORTER_OTLP_ENDPOINT=127.0.0.1:4317
 
 # Integration Test App
 
 from create_flask_app import app, get_flask_app_run_args
 
-# Setup AWS X-Ray Propagator
-
-# Propagators can be set using environment variable: OTEL_PROPAGATORS = aws_xray
-# propagate.set_global_textmap(AwsXRayFormat())
+# NOTE: Resource Detectors cannot be set using auto-instrumentation, they
+# require manually configuring a TracerProvider. See the manual instrumentation
+# Sample App for more information.
+
+# trace.set_tracer_provider(
+#     TracerProvider(
+#         active_span_processor=span_processor,
+#         id_generator=AwsXRayIdGenerator(),
+#         # resource=get_aggregated_resources(
+#         #     [
+#         #         AwsEc2ResourceDetector(),
+#         #     ]
+#         # ),
```

Comment:
Probably don't need the double comment here?

Reply:
Sounds good I will remove!